static/app/views/ddm/chartBrush.tsx

```diff
@@ -276,15 +285,16 @@ const getMetricRange = (params: BrushEndResult, useFullYAxis: boolean): MetricRa
   };
 };
 
-const CHART_HEIGHT = 256;
+const CHART_HEIGHT = 275;
```

Comment:
should that be passed as a prop?

Reply:
yes, i'll i planned on handling it when we enable the enhance in dashboards